### PR TITLE
OpenAI-DotNet 6.5.3

### PR DESCRIPTION
**Pull request by SJMakin**
- Added missing ConfigureAwait to await calls

p.s thanks for making this library - its awesome.

### CODE REVIEW

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
please fix the line endings, update the release notes, and bump the version

**Comment by StephenHodgson** [comment]
@SJMakin did you see any other places we're not using configure await calls?

File: OpenAI-DotNet/OpenAI-DotNet.csproj [hunk 1]
```diff
@@ -17,7 +17,9 @@ More context [on Roger Pincombe's blog](https://rogerpincombe.com/openai-dotnet-
     <RepositoryUrl>https://github.com/RageAgainstThePixel/OpenAI-DotNet</RepositoryUrl>
     <PackageTags>OpenAI, AI, ML, API, gpt-4, gpt-3.5-tubo, gpt-3, chatGPT, chat-gpt, gpt-2, gpt</PackageTags>
     <Title>OpenAI API</Title>
-    <PackageReleaseNotes>Version 6.5.2
+    <PackageReleaseNotes>Version 6.5.3
+- Added missing ConfigureAwait calls to async awaits
```
Comment (StephenHodgson) [inline OpenAI-DotNet/OpenAI-DotNet.csproj:21]:
```suggestion
- Added missing ConfigureAwait to await calls
```

**Review by StephenHodgson — CHANGES_REQUESTED** [review]
Tests failed

File: OpenAI-DotNet/Chat/ChatEndpoint.cs [hunk 2]
```diff
@@ -93,12 +93,12 @@ public async IAsyncEnumerable<ChatResponse> StreamCompletionEnumerableAsync(Chat
...
             };
-            var response = await Api.Client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken);
-            await response.CheckResponseAsync(cancellationToken);
+            var response = await Api.Client.SendAsync(request, HttpCompletionOption.ResponseHeadersRead, cancellationToken).ConfigureAwait(false);
+            await response.CheckResponseAsync(cancellationToken).ConfigureAwait(false);
             await using var stream = await response.Content.ReadAsStreamAsync(cancellationToken).ConfigureAwait(false);
             using var reader = new StreamReader(stream);
 
-            while (await reader.ReadLineAsync() is { } line &&
+            while (await reader.ReadLineAsync().ConfigureAwait(false) is { } line &&
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Chat/ChatEndpoint.cs:101]:
```suggestion
            while (await reader.ReadLineAsync() is { } line &&
```

File: OpenAI-DotNet/Files/FilesEndpoint.cs [hunk 3]
```diff
@@ -165,7 +165,7 @@ public async Task<string> DownloadFileAsync(FileData fileData, string directory,
             }
 
             var filePath = Path.Combine(directory, fileData.FileName);
-            await using var fileStream = new FileStream(filePath, FileMode.OpenOrCreate);
+            await using var fileStream = new FileStream(filePath, FileMode.OpenOrCreate).ConfigureAwait(false);
```
Comment (StephenHodgson) [inline OpenAI-DotNet/Files/FilesEndpoint.cs:168]:
```suggestion
            await using var fileStream = new FileStream(filePath, FileMode.OpenOrCreate);
```

**Comment by SJMakin** [comment]
Apologies for the build issue. Push a change, but looks like I just missed the boat!

**Comment by SJMakin** [comment]
BTW there are some redundant awaits in your overloads in the endpoint classes.

**Comment by SJMakin** [comment]
Thanks again for this lib x

**Pull request merged**